concordium-sdk/src/main/java/com/concordium/sdk/transactions/ConfigureBakerPayload.java

```diff
@@ -0,0 +1,154 @@
+package com.concordium.sdk.transactions;
+
+import com.concordium.sdk.types.UInt16;
+import com.concordium.sdk.types.UInt32;
+import lombok.*;
+
+import java.lang.reflect.Field;
+import java.nio.ByteBuffer;
+
+/**
+ * Payload for configuring a baker.
+ * The [Default] implementation produces an empty configure that will have no
+ * effects.
+ */
+@Getter
+@Builder
+@EqualsAndHashCode
+public class ConfigureBakerPayload {
+    /**
+     * The equity capital of the baker.
+     */
+    private final CCDAmount capital;
+    /**
+     * Whether the baker's earnings are restaked.
+     */
+    private final Boolean restakeEarnings;
+    /**
+     * Whether the pool is open for delegators.
+     */
+    private final Integer openForDelegation;
+    /**
+     * The key/proof pairs to verify the baker.
+     */
+    private final ConfigureBakerKeysPayload keysWithProofs;
+    /**
+     * The URL referencing the baker's metadata.
+     */
+    private final String metadataUrl;
+    /**
+     * The commission the pool owner takes on transaction fees.
+     */
+    private final UInt32 transactionFeeCommission;
+    /**
+     * The commission the pool owner takes on baking rewards.
+     */
+    private final UInt32 bakingRewardCommission;
+    /**
+     * The commission the pool owner takes on finalization rewards.
+     */
+    private final UInt32 finalizationRewardCommission;
+
+    ByteBuffer createNotNullBuffer(byte[] bufferBytes) {
+        val buffer = ByteBuffer.allocate(bufferBytes.length);
+        buffer.put(bufferBytes);
+        return buffer;
+    }
+
+    public byte[] getBitMapBytes() {
+        int bitValue = 0;
+        int it = 1;
+        try {
+            for (Field f : getClass().getDeclaredFields()) {
```

Comment:
Isn't this error prone? Doesn't the order matter? I don't think `getDeclaredFields` returns the fields in any particular order?

Reply:
To maintain the order I have hard coded the fields in the code. Lmk if you have some better way to do it.